Integrations/FortiGate/FortiGate.py

```diff
@@ -429,6 +466,79 @@ def create_firewall_service_request(service_name, tcp_range, udp_range):
     return response
 
 
+def ban_ip(ip_addresses_array, time_to_expire):
+    uri_suffix = 'monitor/user/banned/add_users/'
+
+    payload = {
+        'ip_addresses': ip_addresses_array,
+        'expiry': time_to_expire
+    }
+
+    response = http_request('POST', uri_suffix, {}, json.dumps(payload))
+    return response
+
+
+def ban_ip_command():
+    ip_addresses_string = demisto.args()["ip_addresses"]
+    ip_addresses_array = argToList(ip_addresses_string)
+    time_to_expire = demisto.args().get("expiry")
+    if time_to_expire:
+        time_to_expire = convert_arg_to_int(time_to_expire, "expiry")
+    else:
+        time_to_expire = 0
```

Comment:
add a comment to explain the logic for this default
also, set a default of `0` to the argument so the user will be aware of it

Reply:
@guykeller dont think you understood me here
add `0` as default to the argument in the integration settings

Reply:
@Itay4 Gotcha now, will do